html/data/data-fields.txt

```diff
@@ -10,8 +10,10 @@ Generalities:
 - fields that end with _datetime are dates in the iso8601 format: yyyy-mm-ddThh:mn:ssZ
 - fields that end with _tags are comma separated list of tags (e.g. categories_tags is the set of normalized tags computer from the categories field)
 - fields that end with a language 2 letter code (e.g. fr for French) is the set of tags in that language
-- fields that end with _100g correspond to the amount of a nutriment (in g, or kJ for energy) for 100 g or 100 ml of product
-- fields that end with _serving correspond to the amount of a nutriment (in g, or kJ for energy) for 1 serving of the product
+- fields that end with _100g correspond to the amount of a nutriment (in g, or kJ for energy, or kJ for energy-kj or kcal for energy-kcal) for 100 g or 100 ml of product
```

Comment:
That's in fact not the case.  energy-kcal is in kJ.  

-> (in g, or kJ for energy, energy-kj and energy-kcal)

That's how it has been coded: all energy values are stored in kJ.

Reply:
whaaat ?

Reply:
Just like all weight values are stored in g.